src/diffDates.js

```diff
@@ -1,4 +1,4 @@
-import { DateTime } from "./luxon.js";
+import DateTime from "../node_modules/luxon/src/datetime.js";
```

Comment:
можно попроще написать
```suggestion
import DateTime from "luxon";
```